### ISSUE TITLE
Skip video generation if the test is skipped

### ISSUE THREAD

**Issue by alexandrzavalii** [javierbrea/cypress-fail-fast#104]
I think it would make sense to not generate the video if the test is skipped.

Currently it will generate the videos for all tests even for those that it skipped

**Comment by javierbrea** [comment]
Good idea! :smiley: An investigation is needed in order to know if Cypress exposes a method for stopping the video recording.

**Comment by jpike88** [comment]
is this related to why there might be a 2 second or so delay as each test is skipped?

**Comment by javierbrea** [comment]
Hi @jpike88, I suppose it might be related if you have video recording enabled. I haven't noticed such delay without enabling video recording.

**Comment by javierbrea** [comment]
Hi! From v8.0.0, fail-fast uses `this.skip()` in the Mocha context when it is triggered, so skipped tests follow the normal Cypress/Mocha flow instead of being handled in a special way. Closing this issue, because the plugin should not interfere with Cypress/Mocha’s default behavior when a test is skipped.

As a heads-up, v8.0.0 includes several breaking changes, so if you're upgrading, please check the release notes first: **[Release notes v8.0.0](https://github.com/javierbrea/cypress-fail-fast/releases/tag/v8.0.0)**
